content/200-orm/300-prisma-migrate/300-workflows/10-seeding.mdx

```diff
@@ -385,6 +387,121 @@ psql file.sql
 
 </TabbedContent>
 
+### Seeding your database with `@snaplet/seed`
+
+[`@snaplet/seed`](https://docs.snaplet.dev/seed/getting-started/overview) offers a toolkit designed to understand your database schema,
+enabling you to generate production-accurate data and efficiently seed your database. It use the full power of TypeScript to provide you with
+a type-safe and auto-completed experience to write your seed scripts.
+
+It supports PostgreSQL and SQLite.
+
+1. Begin by setting up a local database using `npx prisma migrate dev`. For instance, consider the following Prisma schema featuring a `User` and `Post` model:
+
+```prisma file=schema.prisma
+model User {
+  id    Int    @id @default(autoincrement())
+  email String @unique
+  name  String
+  posts Post[]
+}
+
+model Post {
+  id        Int     @id @default(autoincrement())
+  title     String
+  content   String
+  published Boolean
+  user      User    @relation(fields: [userId], references: [id])
+  userId    Int
+}
+```
+
+2. Execute `npx @snaplet/seed init` to introspect your database. This command generates the seed client and creates a `seed.ts` file.
+3. Use the generated `@snaplet/seed` client within the `seed.ts` file to compose your seeds.
+
+Here's how to create new users and posts by modifying the initially generated `seed.ts` file:
+
+```ts file=seed.ts
+/**
+ * ! Executing this script will delete all data in your database and seed it with 10 versions.
+ * ! Make sure to adjust the script to your needs.
+ * Use any TypeScript runner to run this script, for example: `npx tsx seed.ts`
+ * Learn more about the Seed Client by following our guide: https://docs.snaplet.dev/seed/getting-started
+ */
+import { createSeedClient } from '@snaplet/seed'
+
+const seed = await createSeedClient()
+
+// Truncate all tables in the database
+await seed.$resetDatabase()
+
+//highlight-start
+// Seed the database with 10 User
+await seed.User((x) =>
```

Comment:
@avallete is this the common syntax for seed? Could we make the `x` slightly more verbose? (what is it actually doing and/or represent)

Reply:
Hey @jharrell, the "x" function allows to create a model `n` times, it reads "seed User times 10", we use this convention because we find it terse, as we need to get the function and call it after that.
I think for Prisma users it would be something like a `createMany`?
We don't mind changing this convention if you think it's clearer with `createMany` or any other name. 👍 
```ts
await seed.User((createMany) => createMany(10, {
    // Create 10 posts for each of those users
    Post: (createMany) => createMany(10),
  })
)
```

Reply:
Yeah, I think `createMany` would be better here, just so users familiar with Prisma can understand what's going on in Snaplet. Would you mind making that change?

Reply:
Done in https://github.com/prisma/docs/pull/5927/commits/1071587f8dbf466c797bd285c8907869a62ad0d8